doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -436,34 +447,78 @@ is_midplane_reachable(self):
 #### 3.1.5 ModuleBase class new APIs
 
 ##### 3.1.5.1 Need for consistent storage and access of DPU reboot cause, state and health
-1.  The smartswitch needs to know the reboot cause for DPUs. Please refer to the CLI section for the various options and their effects when executed on the switch and DPUs. 
-
-* Each DPU will update its reboot cause history in the Switch ChasissStateDB upon boot up. The recent reboot-cause can be derived from that list of reboot-causes.
-* The get_reboot_cause will return the current reboot-cause of the module.
-* For persistent storage of the DPU reboot-cause and reboot-caue-history files use the existing host storage path and mechanism.
-
-#### Schema for REBOOT_CAUSE - switch stateDB
+#### Reboot Cause
```

Comment:
@rameshraghupathy need separate section 
1. DPU reboot cause 
2. NPU reboot cause

Reply:
@prgeor  Done

Reply:
@prgeor Added separate sections for DPU reboot cause and NPU reboot cause